src/main/java/org/spongepowered/common/service/pagination/SpongePaginationBuilder.java

```diff
@@ -105,7 +106,9 @@ public SpongePaginationBuilder(SpongePaginationService service) {
 
     @Override
     public PaginationList build() {
-        checkNotNull(this.contents, "contents");
+        if (this.contents == null) {
+            this.contents = ImmutableList.of();
+        }
```

Comment:
IMO its more likely to be an error to not specify the contents. So this should throw an ISException. But it does not really matter.